src/passes/pass.cpp

```diff
@@ -698,6 +698,7 @@ void PassRunner::handleAfterEffects(Pass* pass, Function* func) {
 int PassRunner::getPassDebug() {
   static const int passDebug =
     getenv("BINARYEN_PASS_DEBUG") ? atoi(getenv("BINARYEN_PASS_DEBUG")) : 0;
+  int unusedVar = 5;
```

Comment:
This is to trigger a clang-tidy error and will not be included in the final commit.